web/src/styles/globals.css

```diff
@@ -2,8 +2,6 @@
   --background: #ffffff;
   --foreground: #171717;
 
-  --grey: color-mix(in srgb, var(--background) 70%, var(--foreground) 30%);
```

Comment:
is there a particular reason for removing this?

Reply:
- Yes, it was causing a test failure as the contrast did not pass the accessibility requirement.
- I have ideas about the use of css variables, so I will open a separate ticket for that. In short though, using variable names like `grey` may not be too helpful as you may as well use the direct `grey` colour built into css. So defining css variables with names like `color-primary`, `color-seconday`, etc. might be better.